test/runner.js

```diff
@@ -80,17 +94,42 @@ files = files.reduce( ( memo, filePath ) => {
 }, [] );
 
 if ( program.nodeTotal > 1 ) {
-	files = files.filter( ( file, index ) => {
+	testFiles = testFiles.filter( ( file, index ) => {
 		return index % program.nodeTotal === program.nodeIndex;
 	} );
 }
 
-files.forEach( setup.addFile );
-
-mocha.addFile( path.join( __dirname, 'load-suite.js' ) );
+let runner = null;
+const runMocha = () => {
+	console.log( chalk.green( 'Running tests at: ' ),
+		chalk.yellow( new Date() ) );
 
-mocha.run( function( failures ) {
-	process.on( 'exit', function() {
-		process.exit( failures ); //eslint-disable-line no-process-exit
+	runner = getMocha().run( () => {
+		runner = null;
```

Comment:
I think we still need `process.on( 'exit', ... )` to give feedback to our Continuos Integration (Circle CI) that tests have failed.